### DISCUSSION TITLE
Newer/higher resolution thermophile cameras

### DISCUSSION THREAD

**Discussion by jdiegmueller** [mKeRix/room-assistant#1138] [General]
Has anyone out there taken a swing at getting the newer/higher resolution thermophile cameras working?  I bought a couple of 32x24 MLX90640 figuring I could adapt the GridEYE implementation pretty easily -- turns out I was wildly wrong.

**Comment by jdiegmueller** [comment]
No one?  Surprised no one else is interested is interested in this.  Would be willing to donatae an MLX90640 towards the cause should someone want to work on this with me.

**Reply by legovaer** [reply]
The MLX90640 was on my radar for quite some time. If you're willing to donate one, I'm happy to have a look at this 👍

**Reply by jdiegmueller** [reply]
Ok, great.  How can I communicate with you privately?  Apparently I'm not savvy enough to figure this out on GitHub. :/
